### PR TITLE
OSDOCS-469: Add OVNKubernetes CNI plug-in tech preview content

### PR DESCRIPTION
**Pull request by jboxman**
- https://jira.coreos.com/browse/OSDOCS-469

### CODE REVIEW

**Comment by jboxman** [comment]
@openshift/team-documentation, because of how types are defined in Go, with OVN SDN in TP for 4.2, the `defaultNetwork` now supports 2 different SDN types. However, the Go type for each configuration has its own name (`openshiftSDNConfig` vs `OVNKubernetesConfig`).

So I wanted to experiment with separating out the configuration object specific to `defaultNetwork` because I don't see another way of presenting this information. (In the future there may be additional SDNs that are configured, such as for Kuryr.)

As a bonus, this CR is displayed as part of customized network installation and for the CNO assembly and uses idef blocks.

Does this presentation make sense or is there a better approach for these configuration objects?

**Comment by jboxman** [comment]
This is complimented by https://github.com/openshift/openshift-docs/pull/15999, so all installs can enjoy OVN SDN.

**Comment by jboxman** [comment]
@openshift/team-documentation, any thoughts on this layout?

- [AWS custom network install - Cluster Network Operator custom resource (CR)](https://osdocs-469--ocpdocs.netlify.com/openshift-enterprise/latest/installing/installing_aws/installing-aws-network-customizations.html#nw-operator-cr_installing-aws-network-customizations) - Installation view of CR. Includes OVN SDN.

- [AWS custom network install - Network configuration parameters](https://osdocs-469--ocpdocs.netlify.com/openshift-enterprise/latest/installing/installing_aws/installing-aws-network-customizations.html#network-customization-config-yaml_installing-aws-network-customizations) - The Tech Preview is dangling at the top; Is there a better place for this?

- [Understanding CNO - Cluster Network Operator custom resource (CR)](https://osdocs-469--ocpdocs.netlify.com/openshift-enterprise/latest/networking/cluster-network-operator.html#nw-operator-cr_cluster-network-operator) - Excluding OVN SDN; Or I could include it but it may need a Tech Preview notice? OVN SDN can't be switched to or configured after install.

There will be a separate KCS with the specifics for this TP.

**Comment by jboxman** [comment]
Hi @zhaozhanqi @anuragthehatter 

This PR adds OVN SDN to the installation docs for AWS. Can you confirm that it's possible to install a new cluster with OVN SDN as the SDN for the `defaultNetwork`?

Thank you!

**Comment by anuragthehatter** [comment]
> Hi @zhaozhanqi @anuragthehatter
> 
> This PR adds OVN SDN to the installation docs for AWS. Can you confirm that it's possible to install a new cluster with OVN SDN as the SDN for the `defaultNetwork`?
> 
> Thank you!
Yes @jboxman ,the cluster can now be brought on OVN by setting "networkType: OVNKubernetes" in install-config.yaml before install and post install defaultNetwork becomes OVNKubernetes in CNO config. Is that what you are looking for?

**Comment by jboxman** [comment]
@anuragthehatter 

Before I can merge this PR, QE needs to verify if the procedure for using OVN SDN as the SDN works.

Thanks!

**Comment by jboxman** [comment]
@openshift/team-documentation, PTAL? Thanks!

File: modules/nw-operator-cr.adoc [hunk 1]
```diff
@@ -120,21 +101,107 @@ allocated and the subnet prefix length assigned to each individual node.
...
+`Multitenant`, `Subnet`, or `NetworkPolicy`. The default value is
+`NetworkPolicy`.
+
+<4> MTU for the VXLAN overlay network. This value is normally configured
+automatically, but if the nodes in your cluster do not all use the same MTU,
+then you must set this explicitly to 50 less than the smallest node MTU value.
+
+<5> The port to use for all VXLAN packets. The default value is `4789`. If you
+are running in a virtualized environment with existing nodes that are part of
+another VXLAN network then you might be required to change this.
```
Comment (ahardin-rh) [inline modules/nw-operator-cr.adoc]: network, then

**Comment by ahardin-rh** [comment]
@jboxman just one minor comment from me; the format looks good!

**Comment by jboxman** [comment]
@ahardin-rh thanks!

**Pull request merged**